### PR TITLE
Admin developer task

### PR DESCRIPTION
**Pull request by pretendWhale**
## Motivation and Context

This removes the creation of a secondary admin user from the `db:admin` task, which is not necessary. It also renames the admin user's last name to 'user', which makes it slightly more consistent with the instructor user.

## Your Changes

**Description**:
Remove creation of second admin, and make the first and last name of the `.admin` user distinct.

**Type of change** (select all that apply):

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] Other (please specify): update rake task to create only one admin user

## Testing

Tested by running `rails db:reset` and inspecting the database afterwards, as well as logging in via the web interface.

## Questions and Comments (if applicable)

Currently the admin user created has the `user_name` `.admin`. we could change this, or we could create a separate task to create a user with the `user_name` `admin`, which might be more intuitive as a `user_name` for an admin user in development and testing.

## Checklist

- [x] I have performed a self-review of my own code.
- [x] I have verified that the pre-commit.ci checks have passed. 
- [ ] I have verified that the CI tests have passed. 
- [x] I have reviewed the test coverage changes reported on Coveralls.

### CODE REVIEW

**Review by david-yz-liu — COMMENTED** [review]
@pretendWhale it seems like there are a few different changes made in this PR, can you give more context about the purpose of each change?

**Comment by pretendWhale** [comment]
@david-yz-liu Sure! The initial goal was just to move the creation of the second admin user out of the `db:admin` task. As far as I can tell, that `test admin` user is primarily for convenience during development (allowing login as `admin` rather than `.admin`).

My initial commit moved the creation of the second user to a new task and added it to the `seeds`, to minimize changes to development. I ended up reconsidering and just removing the second user because it's not actually necessary. I don't have strong feelings about this, and I'm happy to revert back to that.

I also updated the admin user's last name to `user` because I thought it was more consistent with the naming scheme for the users generated by the `instructor` task. I also don't have strong feelings about this, but I noticed we had changed the instructor user's login to 'test instructor' in #5897 so I added it.

File: lib/tasks/users.rake [hunk 1]
```diff
@@ -2,12 +2,6 @@ namespace :db do
   task admin: :environment do
     user = AdminUser.find_or_create
```
Comment (david-yz-liu) [inline lib/tasks/users.rake:3]: Okay, let's do the following:

1. Add a `puts` to log this action, similar to the other user types
2. Delete the `puts user.api_key` on the following line (seems like a leftover from testing or something), and so you won't need the `user` variable any more either.

**Review by david-yz-liu — COMMENTED** [review]
@pretendWhale okay great that all makes sense, thanks. I just left one comment to further simplify the admin creation task.

I'm fine leaving the username as `.admin`, it's just something developers will need to know about, which is fine.

**Review by david-yz-liu — APPROVED** [review]

**Pull request merged**